meadow-client/default.nix

```diff
@@ -50,9 +50,9 @@ in stdenv.mkDerivation {
     ${patchShebangs "node_modules/.bin/"}
 
     mkdir generated
-    mkdir web-common
+    mkdir ../web-common
     cp -R ${psSrc}/* generated/
-    cp -R ${webCommon}/* web-common/
+    cp -R ${webCommon}/* ../web-common/
```

Comment:
Just wondering if it would work if you replaced the `mkdir ... ; cp ...` with `ln -s ${webCommon} ../web-common`?

Reply:
Same question for `psSrc/generated` now I think of it...

Reply:
apparently not, at least I couldn't get it to work